server.js

```diff
@@ -10,6 +11,19 @@ app.set('view engine', 'hbs');
 app.use(express.static(path.join(__dirname, 'public')));
 
 // add your routes here :)
+app.get('/code/:code', (req, res) => {
+
+  const { code } = req.params;
+
+  request(`http://localhost:8080/code/${code}`, (error, response, body)=>{
```

Comment:
I would consider using `fetch` here as it uses promises which are really nice and readable rather than callbacks